docs/detections/rules-ui-create.asciidoc

```diff
@@ -274,15 +274,13 @@ To preview a rule:
 +
 . Select a timeframe of data to preview query results -- *Last hour*, *Last day*, or *Last month* -- from the *Quick query preview* drop-down.
 +
-. Click *Preview results*. A histogram shows the number of alerts you can expect based on the defined rule parameters and historical events in your indices.
-
-A "noise warning" is displayed if the preview generates more than one alert per hour.
-
+. Click *Preview results*. The rule preview shows a histogram and alerts table with the alerts you can expect, based on the defined rule parameters and historical events in your indices. You can view the details of a particular alert by clicking the *View details* button in the alerts table.
++
+NOTE: The preview excludes the effects of rule exceptions and timestamp overrides. Alerts with multiple `event.category` values are counted more than once.
```

Comment:
We stack by `event.category` for every rule type except ML rule previews which are stacked by `host.name`. The note changes for that use case because it has a similar behavior but we probably also wanna include it in docs.

Reply:
Thanks for pointing this out -- please see https://github.com/elastic/security-docs/pull/1838#discussion_r852152019 for revision.